### PR TITLE
feat(order): randomly generate product name and SKU in order item fac…

### PR DESCRIPTION
**Pull request by griest024**
…tory

## PR Checklist
Please check if your PR fulfills the following requirements:

- [x] The commit message follows our guidelines: https://github.com/graycoreio/daffodil/blob/master/CONTRIBUTING.md#commit
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?

```
[ ] Bugfix
[x] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Build related changes
[ ] CI related changes
[ ] Documentation content changes
[ ] Other... Please describe:
```

## What is the current behavior?

The order item factory does not generate fake data for the `sku` and `name` fields.

## What is the new behavior?
The order item factory generates fake data for the `sku` and `name` fields.

## Does this PR introduce a breaking change?
```
[ ] Yes
[x] No
```

## Other information

### CODE REVIEW

**Review by damienwebdev — APPROVED** [review]
lgtm

File: libs/order/testing/src/factories/order-item.factory.ts [hunk 1]
```diff
@@ -20,8 +20,8 @@ export class MockOrderItem implements DaffOrderItem {
   updated_at = faker.date.past();
   product_id = faker.random.number({min: 1, max: 1000});
   parent_item_id = faker.random.number({min: 1, max: 1000});
-  sku = 'sku';
-  name = 'Product Name';
+  sku = faker.random.alphanumeric(20);
```
Comment (damienwebdev) [inline libs/order/testing/src/factories/order-item.factory.ts:23]: The appropriate function name here is `faker.random.alphaNumeric(20)` -- see https://github.com/marak/Faker.js/#api-methods

I'll open up an issue to discuss why we don't have typehinting here when we should.

**Review by damienwebdev — CHANGES_REQUESTED** [review]

File: libs/order/testing/src/factories/order-item.factory.ts (hunk 1 above)
Comment (griest024) [reply libs/order/testing/src/factories/order-item.factory.ts:23]: Maybe we should start a `@types/faker` contribution.

Comment (griest024) [reply libs/order/testing/src/factories/order-item.factory.ts]: fixed

Comment (damienwebdev) [reply libs/order/testing/src/factories/order-item.factory.ts]: I believe one already exists and I presume either we don't use it, or somehow it's not loading correctly.

Comment (griest024) [reply libs/order/testing/src/factories/order-item.factory.ts]: bump

**Review by damienwebdev — APPROVED** [review]
lgtm

**Pull request merged**